announcements/2020-09-02-optional-nginx-lb-recreate-on-upgrade-to-1.8.0+.md

```diff
@@ -0,0 +1,18 @@
+### :zap: Dear Giant Swarm customer :zap:
+
+We'd like to inform you about a known issue when upgrading NGINX ingress controller (IC) optional app to v1.8.0+ on AWS and Azure clusters - NGINX IC LoadBalancer Service may get renamed/recreated with the app upgrade.
```

Comment:
```suggestion
We'd like to inform you about a known issue when upgrading NGINX ingress controller (IC) optional app to v1.8.0+ on *AWS and Azure* clusters - NGINX IC LoadBalancer Service may get renamed/recreated with the app upgrade.
```

Reply:
Fine with making it bold, still I assume this will be sent to AWS and Azure customers only.

Reply:
True, but some customers have both cloud and on prem